README.md

```diff
@@ -79,30 +79,33 @@ The tests are automatically run by Travis CI on every PR and push to
 the repo.  There are two separate sets of tests: `tests/mypy_test.py`
 runs tests against [mypy](https://github.com/python/mypy/), while
 `tests/pytype_tests.py` runs tests against
-[pytype](https://github.com/google/pytype/).  The script runtests.sh
-runs both sets of tests and flake8 over all .pyi files.
+[pytype](https://github.com/google/pytype/).
 
 Both sets of tests are shallow -- they verify that all stubs can be
 imported but they don't check whether stubs match their implementation
 (in the Python standard library or a third-party package).  Also note
 that each set of tests has a blacklist of modules that are not tested
 at all.  The blacklists also live in the tests directory.
 
-To manually run the mypy tests, you need to have Python 3.5 or higher.
+To manually run the mypy tests, you need to have Python 3.5 or higher;
+Python 3.6.1 or higher is recommended.
 Run:
 ```
-$ python3.5 -m venv .venv3
+$ python3.6 -m venv .venv3
 $ source .venv3/bin/activate
-(.venv3)$ pip install -r requirements-tests-py3.txt
+(.venv3)$ pip3 install -r requirements-tests-py3.txt
 ```
-This will install mypy-lang, typed-ast, and flake8. You can then run
-mypy tests and flake8 tests by invoking:
+This will install mypy (you need the latest master branch from GitHub),
+typed-ast, and flake8. You can then run mypy tests and flake8 tests by
+invoking:
 ```
-(.venv3)$ python tests/mypy_test.py
+(.venv3)$ python3 tests/mypy_test.py
 ...
 (.venv3)$ flake8
 ...
 ```
+(Note that flake8 only works with Python 3.6 or higher.)
```

Comment:
Actually, flake8 in 3.5 won't be able to parse typeshed anyway because we're using PEP 526 syntax in stubs now.

@ambv do you think it's still worth it for other use cases to keep flake8-pyi 3.5-compatible?